cli/cmd/delta.go

```diff
@@ -23,32 +23,33 @@ import (
 )
 
 func init() {
-	draftCmd.Flags().StringVarP(&scoreFile, "file", "f", scoreFileDefault, "Source SCORE file")
-	draftCmd.Flags().StringVar(&overridesFile, "overrides", overridesFileDefault, "Overrides file")
-	draftCmd.Flags().StringVar(&extensionsFile, "extensions", extensionsFileDefault, "Extensions file")
-	draftCmd.Flags().StringVar(&apiUrl, "url", apiUrlDefault, "Humanitec API endpoint")
-	draftCmd.Flags().StringVar(&apiToken, "token", "", "Humanitec API authentication token")
-	draftCmd.MarkFlagRequired("token")
-	draftCmd.Flags().StringVar(&orgID, "org", "", "Organization ID")
-	draftCmd.MarkFlagRequired("org")
-	draftCmd.Flags().StringVar(&appID, "app", "", "Application ID")
-	draftCmd.MarkFlagRequired("app")
-	draftCmd.Flags().StringVar(&envID, "env", "", "Environment ID")
-	draftCmd.MarkFlagRequired("env")
+	deltaCmd.Flags().StringVarP(&scoreFile, "file", "f", scoreFileDefault, "Source SCORE file")
+	deltaCmd.Flags().StringVar(&overridesFile, "overrides", overridesFileDefault, "Overrides file")
+	deltaCmd.Flags().StringVar(&extensionsFile, "extensions", extensionsFileDefault, "Extensions file")
+	deltaCmd.Flags().StringVar(&apiUrl, "url", apiUrlDefault, "Humanitec API endpoint")
+	deltaCmd.Flags().StringVar(&uiUrl, "ui", uiUrlDefault, "Humanitec UI")
```

Comment:
Can we pick a better and self-explanatory name for the parameter? For example, `--ui-url`. We can also change `--url` to `--api-url` to avoid confusion. If we keep it as `--ui`, then we are better to rename `--url` to `--api` as well.

Reply:
Changed as suggested - are api-url and ui-url now.